plugins/module_utils/network/iosxr/config/interfaces/interfaces.py

```diff
@@ -205,6 +233,18 @@ def _state_merged(self, want, have):
 
         return commands
 
+    def _state_rendered(self, want):
```

Comment:
@rohitthakur2590 can be part of _state_merged fn. itself at [line](https://github.com/ansible-collections/cisco.iosxr/pull/18/files#diff-98fe0ecae9135809b476b121cbe912a5R230) as:
```suggestion
   if self.state=='rendered':
       commands.extend(self._set_config(interface, dict()))
```
Can you check and let me know if it works as expected for `rendered`.

Reply:
@justjais  yeah, this should work, I could simply add this in merged function, won't be needing separate fund for rendered. Thanks